src/Redmine/Api/AbstractApi.php

```diff
@@ -42,7 +42,7 @@ public function lastCallFailed()
      * @param string $path
      * @param bool   $decodeIfJson
      *
-     * @return string|array|SimpleXMLElement|null
+     * @return string|array|SimpleXMLElement|false|null
```

Comment:
Please remove the `|null` because this will never be the case.

Reply:
`json_decode()` is able to return `null` if the decoded string is "null", but I think the API response will never print that.
Ref: https://3v4l.org/8sXS6.